daft/series.py

```diff
@@ -756,22 +774,27 @@ def from_series(cls: type[SomeSeriesNamespace], series: Series) -> SomeSeriesNam
         ns._series = series._series
         return ns
 
+    def _apply_unary_expr(self, func) -> Series:
+        s = Series._from_pyseries(self._series)
+        return s._apply_unary_expr(func)
+
+    def _apply_binary_expr(self, other: Series, func) -> Series:
+        s = Series._from_pyseries(self._series)
+        return s._apply_binary_expr(other, func)
+
 
 class SeriesFloatNamespace(SeriesNamespace):
     def is_nan(self) -> Series:
```

Comment:
```suggestion
    is_nan = partialmethod(_unary_method, native.is_nan)
```

Reply:
🤯 I had no idea you could do this in python!

Reply:
I'll add this to the pt3 PR instead of here.